example/satosa/pyeudiw_backend.yaml

```diff
@@ -19,11 +19,9 @@ config:
       get_response: '/get-response'
 
   qrcode:
-    size: 100
+    size: 250 #px
     color: '#2B4375'
     expiration_time: 120 # seconds
-    logo_path: 
```

Comment:
logo_path is required, otherwise we don't allow the user to customize the logo within the qrcode